docs/application/native/guides/ui/dali/TOC.md

```diff
@@ -18,6 +18,7 @@
 ### [ImageView](native-app/guides/ui/dali/imageview.md)
```

Comment:
This file ` docs/application/native/guides/ui/dali/TOC.md` is removed on PR 163. So you don't need to update this file anymore.

Reply:
Oh okay. Thank you. Removed the added line.